src/components/ShoppingListItem/ShoppingListItem.js

```diff
@@ -128,8 +130,11 @@ const ShoppingListItem = ({
         </label>
         <button
           type="button"
+          // onClick={toggleDetailView}
```

Comment:
Now we just have to figure out how to get the individual click on the individual panel to open just the one panel!